.circleci/config.yml

```diff
@@ -22,8 +24,28 @@ jobs:
           file: schemas/target/jacoco-reports/jacoco.xml
       - codecov/upload:
           file: generator/target/jacoco-reports/jacoco.xml
+
+  release:
+    docker:
+      - image: circleci/openjdk:8-jdk
+    steps:
+      - checkout
+      # If changing build tools be sure to update GeneratorSetup.md in docs
+      - run: gradle :schemas:release :generator:release
+      - store-artifacts:
+          path: schemas/build/libs
+      - store-artifacts:
+          path: generator/build/libs
+
 workflows:
-  version: 2.1
-  build_and_test: 
+  version: 2
```

Comment:
Is this an intentional downgrade?

Reply:
Yes. https://circleci.com/docs/2.0/configuration-reference/#version-1

2.1 was never correct here.